### DISCUSSION TITLE
The SQLite file created by the Navicat tool is not recognized

### DISCUSSION THREAD

**Discussion by bouyeijiang** [dbeaver/dbeaver#13623] [General]
open the sqlite file not found the table however is has a table

**Comment by uslss** [comment]
@bouyeijiang can you provide a sample database that could help to reproduce an issue?

**Comment by bouyeijiang** [comment]
[db.zip](https://github.com/dbeaver/dbeaver/files/6690324/db.zip)

**Comment by uslss** [comment]
@bouyeijiang unfortunately cannot reproduce it. Connection to database is successfull for me.
![image](https://user-images.githubusercontent.com/31996417/125290734-4ecaa600-e329-11eb-8a3e-924af8c1b791.png)
what dbeaver version do you use? could you provide, e.g. video of the issue?
is there something in the log?
sorry for the late response

**Comment by bouyeijiang** [comment]
![xx](https://user-images.githubusercontent.com/4083760/125384462-8a1db100-e3cb-11eb-918f-fbadcd135368.png)
![zb](https://user-images.githubusercontent.com/4083760/125384493-97d33680-e3cb-11eb-9fa7-ae83161224fe.png)

**Comment by SubOptimal** [comment]
@bouyeijiang
- In your example screenshot the table in Navicat has at least two columns (dian, fx) and the provided example database (`db.db`) has only one column (gz). What's the relation between the two?
- Could it be that the `db.db` file you try to open in dbeaver doesn't contain any table and is not the same as the example you attached to the ticket?
- Could you dump the database with the sqlite3 executable?
e.g. on Linux with the attached `db.db`
```
$ sqlite3 db.db .dump
PRAGMA foreign_keys=OFF;
BEGIN TRANSACTION;
CREATE TABLE IF NOT EXISTS "gz" (
"id"  INTEGER NOT NULL,
PRIMARY KEY ("id")
);
COMMIT;
```
